Compiling the project on a machine with only VS 2017

Running build.ps1 on a machine with only VS 2017 fails as Cake finds the wrong msbuild.exe (http://stackoverflow.com/a/34364698 notes that the msbuild.exe in C:/WINDOWS... is pre-C# 6, and hence does not understand string interpolation).

When I run build.ps1 with -Verbosity Diagnostic it get
```
...
========================================
Build
========================================
Executing task: Build
Executing: "C:/WINDOWS/Microsoft.NET/Framework64/v4.0.30319/MSBuild.exe" /v:minimal /nr:false /p:Configuration="Release" /p:Platform="Any CPU" /target:Build "C:/Temp/nunit-gui/nunit-gui.sln"
Microsoft (R) Build Engine version 4.6.1586.0
[Microsoft .NET Framework, version 4.0.30319.42000]
Copyright (C) Microsoft Corporation. All rights reserved.

  nunit.uikit -> C:\Temp\nunit-gui\bin\Release\nunit.uikit.dll
Views\AddinPages\ExtensionNodeView.cs(92,34): error CS1056: Unexpected character '$' [C:\Temp\nunit-gui\src\nunit-gui\nunit-gui.csproj]
```

Lifting the Cake version to 0.19.1 (from 0.15.2) solves the problem.
```
...
========================================
Build
========================================
Executing task: Build
Executing: "C:/Program Files (x86)/Microsoft Visual Studio/2017/Professional/MSBuild/15.0/Bin/amd64/MSBuild.exe" /v:minimal /nr:false /p:Configuration="Release" /p:Platform="An
y CPU" /target:Build "C:/Temp/nunit-gui/nunit-gui.sln"
Microsoft (R) Build Engine version 15.1.1012.6693
Copyright (C) Microsoft Corporation. All rights reserved.

  nunit.uikit -> C:\Temp\nunit-gui\bin\Release\nunit.uikit.dll
  nunit-gui -> C:\Temp\nunit-gui\bin\Release\nunit-gui.exe
  mock-assembly -> C:\Temp\nunit-gui\bin\Release\mock-assembly.dll
  nunit-gui.tests -> C:\Temp\nunit-gui\bin\Release\nunit-gui.tests.dll
```

I think the problem was solved by cake-build/cake#1369 and related issues.

--- a/tools/packages.config
+++ b/tools/packages.config
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-    <package id="Cake" version="0.15.2" />
+    <package id="Cake" version="0.19.1" />
     <package id="NUnit.ConsoleRunner" version="3.6.1" />
 </packages>

